sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Primitives/MessagingProvider.cs

```diff
@@ -212,5 +212,10 @@ protected internal virtual ServiceBusSessionProcessor CreateSessionProcessor(Ser
             processor.SessionClosingAsync += Options.SessionClosingHandler;
             return processor;
         }
+
+        private static string GenerateCacheKey(string fullyQualifiedNamespace, string entityPath)
+        {
+            return $"{fullyQualifiedNamespace}/{entityPath}";
```

Comment:
Should we consider normalizing case and/or trimming spaces?

Reply:
Hmm, I think the namespace should already be normalized. As for the entity path, we could normalize but I'd prefer to err on the side of keeping it really simple and only using the cache for exact matches.